web-components/src/components/fixed-footer/SaveFooter.tsx

```diff
@@ -106,33 +92,16 @@ const SaveFooter: React.FC<React.PropsWithChildren<Props>> = ({
         </Grid>
 
         <Grid item className={classes.arrows}>
-          {props.onPrev && (
-            <OutlinedButton
-              className={cn(classes.btn, 'mr-10 sm:mr-20')}
-              onClick={props.onPrev}
-            >
-              <ArrowDownIcon
-                fontSize="small"
-                direction="prev"
-                color={theme.palette.secondary.main}
-              />
-            </OutlinedButton>
-          )}
-          <ContainedButton
-            type="submit"
-            className={classes.btn}
-            onClick={props.onSave}
-            disabled={props.saveDisabled}
-          >
-            {saveText}
-          </ContainedButton>
+          <PrevNextButtons
+            saveDisabled={props.saveDisabled}
+            saveText={saveText}
+            onPrev={props.onPrev}
+            onSave={props.onSave}
+          />
         </Grid>
       </Grid>
-      {!hideProgress && (
-        <StyledLinearProgress
-          variant="determinate"
-          value={props.percentComplete}
-        />
+      {!hideProgress && typeof percentComplete !== 'undefined' && (
```

Comment:
I usually prefer using `typeof` because this is safer if the variable hasn't already been declared, it won't throw an error, but I agree in this case, iwe could also use `percentComplete !== undefined` I don't have a strong preference